clients/privacy-center/pages/404.tsx

```diff
@@ -0,0 +1,78 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import NextLink from 'next/link';
+import { Stack, Heading, Box, Text, Button } from '@fidesui/react';
+
+const Custom404 = () => (
```

Comment:
Great attention to detail @elliotbonneville — I don't think this was in the initial spec was it?

Reply:
Came in from the app template!